crates/bifrost/src/read_stream.rs

```diff
@@ -869,49 +882,24 @@ mod tests {
                 assert_eq!(Lsn::from(i), lsn);
             }
 
-            // manually seal the loglet, create a new in-memory loglet at base_lsn=11
-            let raw_loglet = bifrost
-                .inner()
-                .find_loglet_for_lsn(LOG_ID, Lsn::new(5))
+            // seal the loglet and extend with an in-memory one
+            let new_segment_params = new_single_node_loglet_params(ProviderKind::InMemory);
+            bifrost_admin
+                .seal_and_extend_chain(
+                    LOG_ID,
+                    None,
+                    Version::MIN,
+                    ProviderKind::InMemory,
+                    new_segment_params,
+                )
```

Comment:
Should we assert that sealing has actually happened? Now it seems that the test would also pass if `seal_and_extend_chain` would be a no-op.

Reply:
Good call. Let me do that.